x/examples/fetch/main.go

```diff
@@ -42,35 +42,61 @@ func init() {
 func main() {
 	verboseFlag := flag.Bool("v", false, "Enable debug output")
 	transportFlag := flag.String("transport", "", "Transport config")
+	addressFlag := flag.String("address", "", "Address to connecto to. If empty, use the URL authority.")
+	methodFlag := flag.String("method", "GET", "The HTTP method to use.")
```

Comment:
Removing `.` as mentioned above.

```suggestion
	methodFlag := flag.String("method", "GET", "The HTTP method to use")
```